<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="menuWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(150, 150, 150, 255));  
border-right: 2px groove gray; 
}

QWidget::item:hover 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(200, 200, 200, 255)); 
}

QWidget::item:selected 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
}

QWidget::item 
{ 
border-bottom: 2px groove gray; 
padding: 10px; color: white; 
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonMenu1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(255,255,255,150);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/MyFile/Images/menu1.png</normaloff>:/MyFile/Images/menu1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMenu2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/MyFile/Images/menu2.png</normaloff>:/MyFile/Images/menu2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="processWidget">
       <property name="styleSheet">
        <string notr="true">QWidget#processWidget	{
border-image:url(:/MyFile/Images/background.png)
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="processLabelProgress">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelImage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>350</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 5px solid  #f0f0f0;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QComboBox" name="comboBoxProcessOptions">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonProcess">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>Procesar</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonProcessShowAllImages">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>Mostrar Im√°genes</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="pushButtonProcessScan">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/MyFile/Images/scan.png</normaloff>:/MyFile/Images/scan.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>48</width>
                   <height>48</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonProcessLoad">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>60</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);s
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/MyFile/Images/openImage.png</normaloff>:/MyFile/Images/openImage.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>48</width>
                   <height>48</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="varificationWidget">
       <property name="styleSheet">
        <string notr="true">QWidget#varificationWidget	{
border-image:url(:/MyFile/Images/background.png)
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelVerificationScore">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="leftImage" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelLeftImage">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 5px solid  #f0f0f0;</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="pushButtonVeriLeftScan">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);
	border-image:url(:/MyFile/Images/menu1.png)
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	border-radius: 3px;
	border: 1.5px solid white;
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="Resources.qrc">
                    <normaloff>:/MyFile/Images/scan.png</normaloff>:/MyFile/Images/scan.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonVeroLeftLoad">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);
	image: url(:/MyFile/Images/scan.png);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="Resources.qrc">
                    <normaloff>:/MyFile/Images/openImage.png</normaloff>:/MyFile/Images/openImage.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="labelRightImage">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 5px solid  #f0f0f0;</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="pushButtonVeriRightScan">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="Resources.qrc">
                    <normaloff>:/MyFile/Images/scan.png</normaloff>:/MyFile/Images/scan.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonVeriRightLoad">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background-color:rgba(255,255,255,0);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:hover {
    background-color:rgba(255,255,255,50);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 12pt;
	color: white;
}


QPushButton:pressed {
    background-color:rgba(255,255,255,150);
	font-family: helvetica neue;
	font-style: 100;
	font-size: 14pt;
	color: white;
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="Resources.qrc">
                    <normaloff>:/MyFile/Images/openImage.png</normaloff>:/MyFile/Images/openImage.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pushButtonVerificationMatch">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Verificar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statisticsWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionAbrir_Blanco_Negro">
   <property name="text">
    <string>Abrir Blanco/Negro</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
